pontoon/base/static/css/translate.css

```diff
@@ -1432,11 +1432,18 @@ p#sign-in-required > a#sidebar-signin {
 }
 
 #helpers > section ul li > header button.approve:hover:before,
-#helpers > section ul li.translated > header button.approve:before {
-  color: #7BC876;
+#helpers > section ul li.translated > header button.unapprove:before {
   content: "";
 }
 
+#helpers > section ul li > header button.approve:hover:before{
+  color: #7BC876;
+}
+
+#helpers > section ul li > header button.unapprove::before {
+  color: #AAA;
+}
+
 #helpers > section ul li > header button.delete:before {
   content: "";
   margin-left: 5px;
```

Comment:
Approved icon should stay green, because we use this color everywhere in the stats. It should become blue on hover to mark the state after click.

This is the diff to the original file that makes this possible:
https://gist.github.com/mathjazz/bb08e90c4faca5bb8053366f1aad0b7b